README.md

```diff
@@ -1,2 +1,96 @@
+  - Discuss who will do which tasks and when you will target completion
+    - What things should be done together as pairs vs individually?
+  - Organize your tasks/stories into iterations (a.k.a waves)
+- Create a data diagram based on the original data [specification](https://github.com/adagold/ride-share-two)
+
+Once the above is complete, this project:
+
+- Requires a shared repo with your partner as a collaborator
+- Requires you to create a Rails application which
+  - Conforms to Rails conventions on naming and inflection
+  - Is created by using `rails new .` you will create a new rails app _inside_ of the fork folder instead of creating a _new_ folder for your rails app
+  - Uses [Postgresql](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/how-to-use-postgres.md) as it's database.
+
+**Your team shall submit a pull request with a link to your Trello board once you are done with the baseline which shall be reviewed before moving onto implementing the requirements**
+
+## Minimum Requirements
+However far you and your partner get, the application should have, at a minimum, the following features:
+- Seeded data in your database from the original CSV files
+- Multiple, related, Active Record models
+- [RESTful routing](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/mvc-and-restful-routing.md)
+- Deployed application to [Heroku](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/11-deploying-to-heroku.md)
+- A "look and feel" that will make you and your partner happy!
+
+## Things to Keep in Mind
+
+1.  Try to include useful business logic in your model classes.
+1.  You can include nested routes, but avoid routes that are more than 2-levels deep.
+1.  Use Semantic HTML
+1.  Make good use of partial views
+1.  Don't forget validations! (can a trip exist without a driver or passenger...)
+1.  You may not be able to get all the user stories done.  **Prioritize!** 
+
+## User Stories
+
+### Overall
+- From any page I can click a menu entry to create a new passenger.
+- From any page I can click a menu entry to create a new driver
+- From any page I can click a menu entry to view a list of all passengers
+- From any page I can click a menu entry to view a list of all drivers
+
+
+### Viewing Passengers
+
+- On the passenger list page I can click on a passenger to see that passenger's details.
+- On the passenger's details page: 
+    -  I can see a list of trips that passenger went on.
+    - The passenger details page also shows the total amount the passenger has been charged.
+    - Clicking on an individual trip will bring up details for the trip.
+    - On a passenger page I can edit and delete passengers.
+    - When viewing a passenger's details I can add a new trip
+	
+### Investigating Trips
+  -  On a trip detail page I can view details of the trip and links to the driver and the passenger
```

Comment:
You can add a section at the top that says "On the trip detail page: " and shorten each future list item